README.md

```diff
@@ -141,58 +236,173 @@ This a simple example on how config should look like. Please go through README t
 ## Metadata
 
 ### ExternalID
-### Labes
+You can specify *externalId* into metadata to match id from your DB and application
+### Labels
+
+It is possible to specify custom key/value storage into metadata *labels* - object with max 30 properties
+
+it's will be added into application result.
 
+metadata example:
+```js
+const config = {
+  apiUrl: 'YOUR_URL',
+  containerId: 'getid-component',
+  flowName: 'yourFlowName',
+  metadata: {
+    externalId: 'string property to match we your unique id',
+    labels: {
+      'my-custom-meta-name-1': 'custom-value-1',
+      'my-custom-meta-name-2': 'custom-value-2',
+    }
+  }
+}; 
+```
 ## Customization
 
 
 ### Container ID
 This can be anything you want, just make sure for it to match the one being specified in HTML div element.
 
 
-### isPopUp
+### mode
+Sdk can be present as one of two mode: 
+* popup
+* inline
+```js
+const config = {
+  apiUrl: 'YOUR_URL',
+  containerId: 'getid-component',
+  flowName: 'yourFlowName',
+  mode: 'popup'
+};
+```
 
 ### Locale
-
-
-List?
+Define the locale key with alpha-2 country string value to predefine the locale for user
+Example:
+```js
+const config = {
+  apiUrl: 'YOUR_URL',
+  containerId: 'getid-component',
+  flowName: 'yourFlowName',
+  locale: 'en'
+};
+```
 
 
 ### Profile
-Add list availeble fieds for crosschecking
-Add keys with custom fields
-
-
-### Widget object
-
-Example: 
-
-``` js
- const config = {
-   apiUrl: 'YOUR_URL',
-   isPopUp,
- }; 
+We have opportunity to pre-fill the Profile form data by set property *profile* into config:
+example 
+```js
+const config = {
+  apiUrl: 'YOUR_URL',
+  containerId: 'getid-component',
+  flowName: 'yourFlowName',
+  profile: [
+    { category: 'First name', value: 'Jon' },
+    { category: 'Last name', value: 'Dow' },
+    { category: 'Date of birth', value: '2010-10-10' },
+    { category: 'Address', value: 'France le croisan 4-27' },
+    { category: 'Date of expiry', value: '1111-11-11' },
+    { category: 'Date of issue', value: '2021-02-15' },
+    { category: 'Document number', value: '4114414141' },
+    { category: 'Email', value: 'email@email.com' },
+    { category: 'Gender', value: 'male' },
+    { category: 'Nationality', value: 'AZE' },
+    { category: 'Personal number', value: '123123123123' },
+  ],
+};
 ```
+In case when you not specify the Form field but specify this data into profile, this data will send from web sdk to the server and will participate in cross-checking
```

Comment:
Form fields and pre-filled fields will be sent to the server and will be compared with extracted data from a document.